src/system/SystemStats.cpp

```diff
@@ -38,7 +38,7 @@ namespace System {
 namespace Stats {
 
 static const Label sStatsStrings[chip::System::Stats::kNumEntries] = {
-#if CHIP_SYSTEM_CONFIG_USE_LWIP && LWIP_PBUF_FROM_CUSTOM_POOLS
+#if CHIP_SYSTEM_CONFIG_USE_LWIP && defined(LWIP_PBUF_FROM_CUSTOM_POOLS) && LWIP_PBUF_FROM_CUSTOM_POOLS
```

Comment:
As above: this is wrong, though this file does not `#include <lwip/opt.h>` and should.